.cljstyle

```diff
@@ -0,0 +1,18 @@
+{:max-blank-lines 2,
```

Comment:
Best to save this file to `$HOME/.cljstyle` - for now we're not storing this configuration in any of our projects, after that you'll have to git reset to pre-fmt run and run it again - to keep things consistent.

Reply:
done